### PR TITLE
FIX Debian+Ubuntu: install bash-completion if missing

### PR DESCRIPTION
**Pull request by allanlaal**
fixes https://github.com/dokku/dokku/issues/4538

### CODE REVIEW

File: bootstrap.sh [hunk 1]
```diff
@@ -48,12 +48,20 @@ install-requirements() {
         apt-get update -qq >/dev/null
         apt-get -qq -y --no-install-recommends install software-properties-common
       fi
+      if ! dpkg -l | grep -q bash-completion; then
+        apt-get update -qq >/dev/null
+        apt-get -qq -y --no-install-recommends install bash-completion
```
[the remark this answers is not in the document]
Comment (allanlaal) [reply bootstrap.sh:53]: makes sense. its not needed for installing dokku itself

[the remark this answers is not in the document]
Comment (curtishall) [reply bootstrap.sh:53]: Why not just install bash-completion when installing build-essential?  apt will skip bash-completion if it is already installed and not error out.

[the remark this answers is not in the document]
Comment (josegonzalez) [reply bootstrap.sh:53]: We are saying that `bash-completion` is - at minimum - a recommendation for the package. If you don't use the bootstrap script to install Dokku, then you miss the dependency.

**Comment by josegonzalez** [comment]
Closing in favor of #4557 as there hasn't been any movement here.

**Pull request closed without merging**